### PR TITLE
Fix: Github reports security vulnerabilities  #933

### PR DESCRIPTION
**Pull request by gaurav9822**
Upgrade camel and spring-data to resolve the security vulnerabilities issue reported by Github.

### CODE REVIEW

**Comment by gaurav9822** [comment]
@iluwatar Please review the pull request.

**Comment by gaurav9822** [comment]
> 
> 
> @gaurav9822 the build is failing due to the dependency updates

@iluwatar Fixed all issues. Please review.

File: repository/src/main/java/com/iluwatar/repository/App.java [hunk 1]
```diff
@@ -84,17 +85,17 @@ public static void main(String[] args) {
     nasta.setSurname("Spotakova");
     repository.save(nasta);
 
-    LOGGER.info("Find by id 2: {}", repository.findOne(2L));
+    LOGGER.info("Find by id 2: {}", repository.findById(2L));
```
Comment (iluwatar) [inline repository/src/main/java/com/iluwatar/repository/App.java]: `findById` returns an `Optional`, probably we don't want to log that but its contents

File: repository/src/main/java/com/iluwatar/repository/AppConfig.java [hunk 2]
```diff
@@ -134,17 +137,17 @@ public static void main(String[] args) {
     nasta.setSurname("Spotakova");
     repository.save(nasta);
 
-    LOGGER.info("Find by id 2: {}", repository.findOne(2L));
+    LOGGER.info("Find by id 2: {}", repository.findById(2L));
```
Comment (iluwatar) [inline repository/src/main/java/com/iluwatar/repository/AppConfig.java]: Returns an `Optional`..

**Comment by iluwatar** [comment]
Thanks @gaurav9822 for the security fixes :policeman:

**Pull request merged**